tests/warp/warp_test.go

```diff
@@ -154,7 +165,53 @@ var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {
 		log.Info("Creating ethclient for blockchainB", "wsURI", chainBWSURI)
 		chainBWSClient, err = ethclient.Dial(chainBWSURI)
 		gomega.Expect(err).Should(gomega.BeNil())
+	})
+
+	// Deploy the test warp contracts to subnets A & B
+	ginkgo.It("Deploy Test Warp Contracts", ginkgo.Label("Warp", "DeployWarp"), func() {
+		ctx := context.Background()
+
+		auth, err := bind.NewKeyedTransactorWithChainID(fundedKey, chainID)
+		gomega.Expect(err).Should(gomega.BeNil())
 
+		//deploy on subnetA
```

Comment:
```suggestion
		// deploy warp contracts on subnetA
```

we should add one character of whitespace between `//` and the actual comment

Reply:
Sounds good!